test/integration/integration.express.test.js

```diff
@@ -0,0 +1,107 @@
+const path = require('path');
+require('dotenv').config({ path: path.join(__dirname, '../../.env') });
+
+const request = require('supertest');
+const fs = require('fs');
+
+//load data samples
+const filePathTweets = path.join(__dirname, '../..', '/public/data/tweets.txt');
+const sampleTextTweets = fs.readFileSync(filePathTweets, 'utf-8');
+
+const filePathReview = path.join(__dirname, '../..', '/public/data/review.txt');
+const sampleTextReview = fs.readFileSync(filePathReview, 'utf-8');
+
+const filePathEmail = path.join(__dirname, '../..', '/public/data/personal-email.txt');
+const sampleTextEmail = fs.readFileSync(filePathEmail, 'utf-8');
+
+const describeIfSkip = process.env.TONE_ANALYZER_IAM_APIKEY || process.env.TONE_ANALYZER_USERNAME ? describe : describe.skip;
```

Comment:
We are adding a new variable here and we are only using it once. Whereas the previous implementation wouldn't even go to `describe` if credentials are not valid. Do we really need to do it this way?

Reply:
Good point @mamoonraja. This approach shows that the test exists but it was skipped while the previous approach didn't show that a test was there because the `describe` was ignore

Reply:
yup :)